### ISSUE TITLE
Unit tests for the challenges

### ISSUE THREAD

**Issue by emreb** [rogy/smr-challenge#14]
There shoudl be unit tests for the case that it should be able to pass.

Fro example for the clock test, if someone just said return 50; they would have been fine.

We should improve these for the next SMR and potentially to have this site hosted.
IF someone is willing to hack and see things for themselves, that is good too.

**Comment by rogy** [comment]
agreed, it should be useful for other recruitment events  ike Uni fairs as well, I will try and spend some time making it easier to add more challenges/test cases and make it more of an app instead of a bunch of separate pages. Will add it to my side projects list.

**Comment by emreb** [comment]
sweet. thanks

**Comment by emreb** [comment]
Here are the unit tests for clock hands to make sure it runs fine.

12:00, 00:00 10:10  14:50
23:00, 22:10, 2:50

**Comment by emreb** [comment]
This is now done for the clock one. 
Someone, some day should do it for the other two. bc right now one can just return the right answer.
